CONTRIBUTING.md

```diff
@@ -70,22 +70,30 @@ helper methods.
 
 #### How to enable and configure
 
-* Create a folder in your project called `.publicApi` with the frameworks that
+1. Create a folder in your project called `.publicApi` with the frameworks that
   as folders you target.
-* Create two files called `PublicAPI.Shipped.txt` and `PublicAPI.Unshipped.txt`
+2. Create two files called `PublicAPI.Shipped.txt` and `PublicAPI.Unshipped.txt`
    in each framework that you target.
-* Add the following lines to your csproj:
+3. Add the following lines to your csproj:
```

Comment:
Let's remove this step.

Everything should be handled automatically across the repo:

* https://github.com/open-telemetry/opentelemetry-dotnet/blob/37535a5607ee7e4056c0e274ec01d1e0111a64be/build/Common.prod.props#L49-L50

* https://github.com/open-telemetry/opentelemetry-dotnet/blob/37535a5607ee7e4056c0e274ec01d1e0111a64be/build/Common.prod.props#L102-L109